src/azure-cli/azure/cli/command_modules/netappfiles/custom.py

```diff
@@ -318,6 +318,13 @@ def _build_arguments_schema(cls, *args, **kwargs):
         # The API does only support setting Basic and Standard
         args_schema.network_features.enum = AAZArgEnum({"Basic": "Basic", "Standard": "Standard"}, case_sensitive=False)
 
+        # For backwards compat, in next breaking change window, those will be removed
+        args_schema.is_restoring = AAZBoolArg(
+            options=["--is-restoring"],
+            arg_group="Properties",
+            help="Restoring. Deprecated, do not use will be removed in future version.",
```

Comment:
You can follow this guide line to add breaking change information for those arguments https://github.com/Azure/azure-cli/blob/dev/doc/breaking_change_detection_guidelines.md

Reply:
added `_breaking_change.py`